src/ert/scheduler/lsf_driver.py

```diff
@@ -107,19 +108,24 @@ async def submit(
             stderr=asyncio.subprocess.PIPE,
         )
         stdout, stderr = await process.communicate()
-        try:
-            job_id = (
-                stdout.decode("utf-8")
-                .strip()
-                .replace("<", "")
-                .replace(">", "")
-                .split()[1]
-            )
-        except IndexError as err:
+        if process.returncode:
             logger.error(
-                f"Command \"{' '.join(bsub_with_args)}\" failed with error message: {stderr.decode()}"
+                f"Command \"{' '.join(bsub_with_args)}\" failed with "
+                f"returncode {process.returncode} and error message: {stderr.decode() or '<empty>'}"
+            )
+            raise RuntimeError(stderr.decode())
```

Comment:
Shall we do `RuntimeError(f"Command \"{' '.join(bsub_with_args)}\" failed with "
                f"returncode {process.returncode} and error message: {stderr.decode() or '<empty>'}")`?

Reply:
I guess my approach makes the error message end up in potentially two log files, while going to RuntimeError directly makes it go only one place (maybe not predetermined where it ends). 

I guess my thought was that it is nice when debugging to have this in the `jobqueue-` log file because that is a file that you most likely are looking at in any case.